airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -134,12 +138,17 @@ def client(self):
 
         return session.client(service_name="secretsmanager", **self.kwargs)
 
-    def _format_uri_with_extra(self, secret, conn_string):
+    @staticmethod
+    def _format_uri_with_extra(secret, conn_string):
         try:
             extra_dict = secret['extra']
         except KeyError:
             return conn_string
-        conn_string = f"{conn_string}?{urlencode(extra_dict)}"
+
+        extra = ast.literal_eval(
+            extra_dict
+        )  # json.loads doesn't work because you can have unquoted booleans values
+        conn_string = f"{conn_string}?{urlencode(extra)}"
```

Comment:
I don't think we need this @JavierLopezT

Reply:
Yes, that is the bug that impeeded to release amazon provider with the others. Extra_dict is a string, so we have to turn it into a dict before urlencode

Reply:
I have added a little clarification in the docs